packages/biometric-ed25519/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@near-js/biometric-ed25519",
+  "description": "JavaScript library to handle webauthn and biometric keys",
+  "version": "1.1.0",
```

Comment:
Should be `0.0.1`

Reply:
Just thought that the version should be kept same with near-api-js package

Reply:
One benefit we get from the monorepo is independent versioning across packages. Our tooling ensures that the `workspace:*` references figure out the correct version to use at publish time, granting us more freedom to make breaking changes while maintaining package isolation.